### ISSUE TITLE
VS debug console does not debug Destructuring assignments

### ISSUE THREAD

**Issue by gibbok** [Microsoft/vscode#16897]
- VSCode Version: 1.7.2 
- OS Version: Windows 8.1
- Node: v6.9.1

Steps to Reproduce:

1. 
Create an index.js file containing the following code:

```
let [,, a,b] = [0,1,2,3];
console.log(a);
console.log(b);
```

2. 

Use the default created launch.json for debuggin in VS debug console, example of launch.json

   ```
     {
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceRoot}\\index.js",
            "cwd": "${workspaceRoot}",
            "args": [ "/test.css", "/test.json" ]
        },
```
3. 

Start debugging using VS debug console ("Launch Program"), at first line an  error is thrown
`
TypeError: Cannot read property 'next' of undefined`

Basically I am not able to debug a Destructuring assignment

**Comment by weinand** [comment]
Please try our new experimental node debug by setting the `type` attribute in the launch config to `node2`.

**Comment by weinand** [comment]
dup of https://github.com/Microsoft/vscode/issues/13216
